tools/spelling_skip_files.txt

```diff
@@ -1,2 +1,4 @@
 OWNERS.md
 test/extensions/filters/http/wasm/test_data/metadata_cpp.wat
+examples/wasm/envoy_filter_http_wasm_example.wat
+examples/wasm/envoy_filter_http_wasm_example.wasm
```

Comment:
Nit: `check_spelling` complains only about the `.wat` file.

Side-note: we could probably just use a `*.wat` regex here to future-proof it.

Reply:
Done.